rules/REQUEST-949-BLOCKING-EVALUATION.conf

```diff
@@ -218,19 +218,25 @@ SecRule TX:BLOCKING_INBOUND_ANOMALY_SCORE "@ge %{tx.inbound_anomaly_score_thresh
     tag:'anomaly-evaluation',\
     tag:'OWASP_CRS',\
     ver:'OWASP_CRS/4.0.1-dev',\
+    setvar:'tx.inbound_early_blocking_evaluated=1',\
     chain"
-    SecRule TX:EARLY_BLOCKING "@eq 1"
+    SecRule TX:BLOCKING_INBOUND_ANOMALY_SCORE "@ge %{tx.inbound_anomaly_score_threshold}" \
+        "chain"
+        SecRule TX:EARLY_BLOCKING "@eq 1" "setvar:'tx.inbound_blocking_evaluated=1',setvar:'tx.outbound_early_blocking_evaluated=1',setvar:'tx.outbound_blocking_evaluated=1'"
 
 # always check threshold in phase 2
-SecRule TX:BLOCKING_INBOUND_ANOMALY_SCORE "@ge %{tx.inbound_anomaly_score_threshold}" \
+SecAction \
     "id:949110,\
     phase:2,\
     deny,\
     t:none,\
     msg:'Inbound Anomaly Score Exceeded (Total Score: %{TX.BLOCKING_INBOUND_ANOMALY_SCORE})',\
     tag:'anomaly-evaluation',\
     tag:'OWASP_CRS',\
-    ver:'OWASP_CRS/4.0.1-dev'"
+    ver:'OWASP_CRS/4.0.1-dev',\
+    setvar:'tx.inbound_blocking_evaluated=1',\
+    chain"
+    SecRule TX:BLOCKING_INBOUND_ANOMALY_SCORE "@ge %{tx.inbound_anomaly_score_threshold}" "setvar:'tx.outbound_early_blocking_evaluated=1',setvar:'tx.outbound_blocking_evaluated=1'"
```

Comment:
Why would you set outbound_blocking_evaluated=1 in 949110?

Reply:
Because rest of the blocking rules are skipped in a case a request is bloked.